blog/not-so-basic-base-r-functions/index.qmd

```diff
@@ -56,29 +62,19 @@ a
 print(f2("This prints because of print()"))
 ```
 
-If you're writing functions, then `invisible()` can be very helpful. In fact, many of your favorite functions use `invisible()`. For example, `readr::write_csv()` [invisibly returns](https://github.com/tidyverse/readr/blob/e4d640ace3bb6ae095a1114158854a38a1da3a30/R/write.R#L214) the data frame that was saved.
-
-* In the example below, using {purrr} and `tidytab::tab2()` together results in two `NULL` list items that I do not need. 
-
-First, let's install the required R packages. This should be quick, not enough time to go get a cup of coffee! Installing purrr from the [webR binary repository](https://docs.r-wasm.org/webr/latest/packages.html) is as easy as calling `webr::install()`.
+Let's try another example. Run the chunk below to install the purrr and tidytab packages. Installing the CRAN version of purrr from the webR binary repository is as easy as calling `webr::install()`. The tidytab package is compiled into a WebAssembly binary on [R-universe](https://gvelasq.r-universe.dev/tidytab#) and needs the `repos` argument to find it. `mount = FALSE` is due to a bug in the Firefox WebAssembly interpreter. If you're not using Firefox, then feel free to try without that argument! (Note: this might take a few seconds.)
```

Comment:
Suggest to change to:

>`mount = FALSE` is to circumvent a bug in the Firefox WebAssembly interpreter. If you're not using Firefox, I suggest you try the code below with `mount = TRUE` (Note: this might take a few seconds, and will take longer with `mount = FALSE`).